libs/ui/src/admin/ModelForm/ModelCreateForm.tsx

```diff
@@ -200,6 +207,48 @@ export const ModelCreateForm = <T extends StrapiModel>({
             )
           }
 
+          if (field.type === 'number' && field.name === 'price') {
```

Comment:
We can add `price` type when want to display currency suffix

```suggestion
          if (field.type === 'price') {
```